apps/examples/testcase/le_tc/kernel/tc_pthread.c

```diff
@@ -1533,13 +1460,9 @@ int pthread_main(void)
 	tc_pthread_pthread_cond_broadcast();
 	tc_pthread_pthread_cond_init_destroy();
 	tc_pthread_pthread_set_get_schedparam();
-	tc_pthread_pthread_key_create_set_getspecific();
+	tc_pthread_pthread_key_create_set_getspecific_delete();
```

Comment:
I think that TC name should be changed like below, because this TC covers pthread_key_create, delete and pthread_get,setspecific.
```
tc_pthread_pthread_key_create_delete_set_getspecific
```